core/core.go

```diff
@@ -53,7 +53,7 @@ const kSyncAccountFreq = time.Hour
 type InitConfig struct {
 	Account         *keypair.Full
 	PinCode         string
-	RepoPath        string
+	BaseRepoPath    string
```

Comment:
Now called `BaseRepoPath` to denote that it's not the actual repo path, but used as a base location. Actual location is determined by account seed.